dot/network/service.go

```diff
@@ -414,47 +417,56 @@ func (s *Service) logPeerCount() {
 	}
 }
 
-func (s *Service) publishNetworkTelemetry(done <-chan struct{}) {
+func (s *Service) publishTelemetry(cancel <-chan struct{}) {
 	ticker := time.NewTicker(s.telemetryInterval)
 	defer ticker.Stop()
 
 	for {
+		s.sentBandwidthTelemetry()
+		err := s.sendBlockIntervalTelemetry()
+		if err != nil {
+			logger.Warnf("failed to sent block interval telemetry: %s", err)
+			continue
```

Comment:
So if we fail, we instantly try again; should we instead wait for the next tick? Or at least some backoff? I'm just thinking for example if the network goes down, this loop might get got

Reply:
yeah, this should just fall through to the `select` statement